pymc_marketing/mmm/budget_optimizer.py

```diff
@@ -50,7 +50,7 @@ class BudgetOptimizer(BaseModel):
         The adstock class.
     saturation : SaturationTransformation
         The saturation class.
-    num_days : int
+    periods : int
         The number of days.
```

Comment:
Number of days or the number of periods?  Also, why `periods` and not `num_periods`?  The latter seems less ambiguous.

Reply:
Okay trying to be more explicit ✌🏻